server/server.ts

```diff
@@ -14,6 +16,86 @@ const PORT = process.env.PORT || 8000;
 const app = express();
 app.use(express.json());
 app.use(express.static('../client'));
+app.use(require('morgan')('combined'));
+
+// From Passport example
+app.use(require('cookie-parser')());
+app.use(require('body-parser').urlencoded({ extended: true }));
+app.use(require('express-session')({
+  secret: 'keyboard cat', // TODO ?
+  resave: true,
+  saveUninitialized: true,
+}));
+
+app.use(passport.initialize());
+app.use(passport.session());
+
+
+passport.use(new Strategy({
+    clientID: process.env['FACEBOOK_CLIENT_ID'],
+    clientSecret: process.env['FACEBOOK_CLIENT_SECRET'],
```

Comment:
🛑 

- [ ] Add to README
- [ ] Migrate locally to a new fb app (probably have separate ones for local and prod?)